src/types.js

```diff
@@ -0,0 +1,9 @@
+export type DOMContainer = Element | Document;
+export type Instance = Element;
+export type TextInstance = Text;
```

Comment:
I think for these i'd be better to use the DOM types until we are adding propeties or something to them. I believe some of these names in the React are more due to legacy than accuracy. I think that seeing `Element` is clearer indication of waht the thing is vs the more vague "instance" and TextInstance, etc

Reply:
Sure. I'll make the change: Instance -> Element and TextInstance -> Text